server/routes/settings.js

```diff
@@ -72,6 +72,20 @@ router.get("/:email", jwtVerify, (req, res, next) => {
       next("User settings doesn't exist!");
     }
   })
+});
+
+router.get("/:email/mentions", jwtVerify, (req, res, next) => {
+  SettingsModel.findOne({ email: req.params.email }).populate('companies').exec(async (err, settings) => {
+    if (settings) {
+      let companyNames = [];
+      settings.companies.forEach(company => companyNames.push(company.name));
+      const interface = new Interface();
+      const mentions = await interface.getAllMentions(companyNames);
+      res.send({ success: true, settings, mentions });
+    } else {
+      next("User settings doesn't exist!");
+    }
+  })
```

Comment:
Route added to grab ALL mentions, now the /:email route just gets newest mentions (name should be changed to reflect that later)